### PR TITLE
docs: disable unsupported `-doc-canonical-base-url` for Scala 2.11

### PR DESCRIPTION
**Pull request by jrudolph**
Follow up to #2662.

### CODE REVIEW

File: project/Doc.scala [hunk 1]
```diff
@@ -50,7 +51,7 @@ object Scaladoc extends AutoPlugin {
       docs
     })
 
-  def scaladocOptions(ver: String, base: File, plugins: Seq[File]): List[String] = {
+  def scaladocOptions(scalaBinaryVersion: String, ver: String, base: File, plugins: Seq[File]): List[String] = {
     val urlString = GitHub.url(ver) + "/€{FILE_PATH_EXT}#L€{FILE_LINE}"
```
Comment (ennru) [inline project/Doc.scala]: This needs to be reverted as well for pre-Scala 2.12.9.

Comment (jrudolph) [reply project/Doc.scala]: Ah, indeed, will do.

Does it work with 2.13.0 and was backported to 2.12.9?

Comment (ennru) [reply project/Doc.scala]: Exactly. It is 2.12.9+

File: project/Doc.scala [hunk 2]
```diff
@@ -50,19 +51,22 @@ object Scaladoc extends AutoPlugin {
       docs
     })
 
-  def scaladocOptions(ver: String, base: File, plugins: Seq[File]): List[String] = {
-    val urlString = GitHub.url(ver) + "/€{FILE_PATH_EXT}#L€{FILE_LINE}"
+  def scaladocOptions(scalaBinaryVersion: String, ver: String, base: File, plugins: Seq[File]): List[String] = {
+    val urlString = GitHub.url(ver) +
+                    // supported from Scala 2.12.9
+                    (if (scalaBinaryVersion != "2.11") "€{FILE_PATH_EXT}#L€{FILE_LINE}" else "€{FILE_PATH}.scala")
```
Comment (jrudolph) [inline project/Doc.scala]: I removed an extra `/` at the beginning I noticed in the generated docs. Will save 1 byte per documented class.

File: project/Doc.scala (hunk 1 above)
Comment (jrudolph) [reply project/Doc.scala]: :+1:

Comment (jrudolph) [reply project/Doc.scala]: Great, thanks.

**Pull request merged**